### ISSUE TITLE
Python 3.11 support?

### ISSUE THREAD

**Issue by hoodmane** [pyb2d/pyb2d#62]
It would be nice to have a release with Python 3.11 support.

**Comment by DerThorsten** [comment]
Hi Hood,

Once https://github.com/pyb2d/pyb2d/pull/63 lands 3.11 should be supported. Locally it works already like a charm

**Comment by DerThorsten** [comment]
closed via #63
